skt/kerneltree.py

```diff
@@ -180,58 +180,24 @@ def get_commit_details(self, ref=None, show_format="%H"):
 
     def get_commit_date(self, ref=None):
         """
-        Get the committer date of the commit pointed at by the specified
-        reference, or of the currently checked-out commit, if not specified.
-
+        Get the commit date of the commit specified by an optional ref.
         Args:
-            ref:    The reference to commit to get the committer date of,
-                    or None, if the currently checked-out commit should be
-                    used instead.
+            ref: A reference to a commit in a git repo to be inspected.
```

Comment:
I like how you shortened the description, but please see if you could still mention the default and what `None` would mean.

Reply:
Fixed.